src/index.js

```diff
@@ -118,33 +118,35 @@ function check(autoCheckElement: AutoCheckElement) {
   }
   autoCheckElement.dispatchEvent(new CustomEvent('loadstart'))
   performCheck(input, body, src)
-    .then(data => {
+    .then(message => {
       autoCheckElement.dispatchEvent(new CustomEvent('load'))
-      const message = data ? data.trim() : null
       if (autoCheckElement.required) {
         input.setCustomValidity('')
       }
       input.dispatchEvent(new CustomEvent('auto-check-success', {detail: {message}, bubbles: true}))
     })
     .catch(error => {
+      let validity = 'Something went wrong'
+
+      if (error.statusCode === 422 && error.responseText) {
+        if (error.contentType.includes('application/json')) {
+          validity = error.responseText.text
+        }
+        if (error.contentType.includes('text/plain')) {
```

Comment:
Perhaps we can just fallback to using whatever the response text is if one exist?